akka-actor-typed/src/main/scala/akka/actor/typed/internal/Restarter.scala

```diff
@@ -56,6 +54,16 @@ import akka.actor.typed.scaladsl.Behaviors
 
   protected def loggingEnabled: Boolean
 
+  /**
+   * Invoked when the actor is created (or re-created on restart) this is where a restarter implementation
+   * can provide logic for dealing with exceptions thrown when running any actor initialization logic (undeferring).
+   *
+   * FIXME: there is something with nested supervisors and init/undefer that I'm missing here, figure out what
```

Comment:
still relevant?

Reply:
I think I explored that and realised it is fine, just forgot to remove the comment. Will do that and rebase.